src/images/common/github_icon.svg

```diff
@@ -0,0 +1,25 @@
+
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"

```

Comment:
Can I be really annoying and ask you to clean up this SVG a little bit? :D
Things like removing the version, id, xmlns:xlink, x, y, and xml:space and then making sure for the path that everything is on one line. Sorry if this is very picky :D :D :D